### PR TITLE
Create `.gitattributes`

### LINKED ISSUE #41: Add .gitattribute specifying what we should use for line ending etc..
see https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/748

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
as discussed in the meeting

closes #41

### CODE REVIEW

**Comment by IndrajeetPatil** [comment]
Need to wait (cf. https://github.com/Open-Systems-Pharmacology/OSPSuite.RUtils/issues/41#issuecomment-1027787539)

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+*.rdb   binary
+*.rds   binary
+*.Rd    text
+*.Rdx   binary
+*.Rmd	  text
+*.R  	  text
+*.Rproj text
+*.[Rr]md   linguist-detectable
+
+R text eol=crlf
```
Comment (msevestre) [inline .gitattributes:17]: no this line should be on 13

*.R text eol=crlf

**Review by msevestre — CHANGES_REQUESTED** [review]

File: .gitattributes (hunk 1 above)
Comment (msevestre) [reply .gitattributes:17]: with the * at the beginning

Comment (IndrajeetPatil) [reply .gitattributes:17]: Done.

Comment (msevestre) [reply .gitattributes:17]: so line 17 should go :)

**Comment by Yuri05** [comment]
2 questions regarding this:

1. **\*.RData** is included in **.gitignore** (which is OK). Why do we need it in **.gitattributes**?
2. **.gitignore** has only **\*.RData** but not **\*.Rdata**. Should we add **\*.Rdata** to **.gitignore** as well?

@IndrajeetPatil @msevestre

**Comment by msevestre** [comment]
>Why do we need it in .gitattributes? 

Probably some people save their RData in repo for examples etc... so this should not be changesd as well. It's not wrong to have it since we are using a template used by many

>gitignore has only *.RData but not *.Rdata. Should we add *.Rdata to .gitignore as well?

it's not *.RData but the file called .RData

 ```
C:\dev\OSPSuite-R (develop -> upstream)
λ touch toto.Rdata

C:\dev\OSPSuite-R (develop -> upstream)
λ git s
?? toto.Rdata

C:\dev\OSPSuite-R (develop -> upstream)
λ touch tutu.RData

C:\dev\OSPSuite-R (develop -> upstream)
λ git s
?? toto.Rdata
?? tutu.RData
```

**Pull request merged**